plugins/action/shutdown.py

```diff
@@ -153,6 +156,9 @@ def perform_shutdown(self, task_vars, distribution):
         shutdown_result = {}
         shutdown_command = self.get_shutdown_command(task_vars, distribution)
         shutdown_command_args = self.get_shutdown_command_args(distribution)
+        if shutdown_command == 'systemctl poweroff':
+            display.vvv('attempting to shutdown using systemd poweroff command')
+            shutdown_command_args = ''
```

Comment:
This seems like a hack. I would combine `get_shutdown_command` and `get_shutdown_command_args` into one function that returns both `shutdown_command` and `shutdown_command_args`, and let it handle this.

Reply:
It is a hack, will fix.